### PR TITLE
Feature/portfolio layout

### PR DESCRIPTION
**Pull request by vlad-elagin**

### CODE REVIEW

File: components/portfolio/works.js [hunk 1]
```diff
@@ -19,16 +19,15 @@ const Works = ({ works }) => (
                   <ul className="portfolio__item-categories">
                     {
                       work.category.main && work.category.main !== 'Other'
-                        ? work.category.main.map((category) => (<li className="portfolio__item-category" key={category}>{category}</li>))
+                        ? work.category.main.map((category) => (<span className="tag" key={category}>{category}</span>))
```
Comment (dmytropanontko) [inline components/portfolio/works.js]: why did you change it?

File: styles/pages/portfolio.scss [hunk 2]
```diff
@@ -1,141 +1,250 @@
...
+	padding: 0 30px;
+	position: relative;
+	@media (max-width: 500px) {
+		padding: 40px 20px;
+	}
 }
+
 .portfolio__header {
-   margin-bottom: 50px;
+	@media (max-width: 574.98px) {
```
Comment (dmytropanontko) [inline styles/pages/portfolio.scss]: Why do you use so different points? Please talk with UI/UX designer and make variables for these points)

File: styles/pages/portfolio.scss [hunk 3]
```diff
@@ -1,141 +1,250 @@
...
-.portfolio__background {
-   min-height: 100vh;
-   position: absolute;
-   left: 0;
-   right: 0;
-   top: 0;
-   bottom: 0;
-   z-index: -1;
-   background: linear-gradient(135deg, #071e40 0%, #662938 80%);
+	padding: 0 30px;
```
Comment (dmytropanontko) [inline styles/pages/portfolio.scss]: you changed indent

File: styles/pages/portfolio.scss [hunk 4]
```diff
@@ -1,141 +1,250 @@
...
+
+		@media (max-width: 767px) {
+			width: 100%;
+		}
+
+		@media (min-width: 768px) and (max-width: 1023px) {
+			width: calc(50% - 32px);
+		}
+
+		@media (min-width: 1024px) {
```
Comment (dmytropanontko) [inline styles/pages/portfolio.scss]: it's better when you use one style of `@media` only `max-width` or `min-width`

File: components/portfolio/works.js (hunk 1 above)
Comment (vlad-elagin) [reply components/portfolio/works.js]: In order to reuse existing `.tag` styles. But I came to create separate class scoped to portfolio page. I'll fix it.

File: styles/pages/portfolio.scss (hunk 2 above)
Comment (vlad-elagin) [reply styles/pages/portfolio.scss]: Those was copypasted from Anton's branch for redesigning blog. I'll use breakpoint mixin instead.
